deploy/kubernetes/setup-sumologic.yaml.tmpl

```diff
@@ -104,16 +104,10 @@ data:
         collector_id = sumologic_collector.collector.id
     }
   
-    resource "kubernetes_namespace" "sumologic_collection_namespace" {
-      metadata {
-        name = var.namespace_name
-      }
-    }
-  
     resource "kubernetes_secret" "sumologic_collection_secret" {
       metadata {
         name = "sumologic"
-        namespace = kubernetes_namespace.sumologic_collection_namespace.metadata[0].name
+        namespace = var.namespace_name
```

Comment:
Looks like base of the branch is not up to date with master

Reply:
Good catch!